src/test/java/subway/StationAcceptanceTest.java

```diff
@@ -26,42 +31,95 @@ public class StationAcceptanceTest {
     @DisplayName("지하철역을 생성한다.")
     @Test
     void createStation() {
-        // when
-        Map<String, String> params = new HashMap<>();
-        params.put("name", "강남역");
+        final String STATION = "강남역";
 
-        ExtractableResponse<Response> response =
-                RestAssured.given().log().all()
-                        .body(params)
-                        .contentType(MediaType.APPLICATION_JSON_VALUE)
-                        .when().post("/stations")
-                        .then().log().all()
-                        .extract();
+        // when
+        ExtractableResponse<Response> response = 지하철역_생성(STATION);
 
         // then
         assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
 
         // then
         List<String> stationNames =
                 RestAssured.given().log().all()
-                        .when().get("/stations")
+                        .when().get(END_POINT)
                         .then().log().all()
-                        .extract().jsonPath().getList("name", String.class);
-        assertThat(stationNames).containsAnyOf("강남역");
+                        .extract().jsonPath().getList(KEY_NAME, String.class);
+        지하철역_이름_검증(STATION, stationNames);
     }
 
     /**
      * Given 2개의 지하철역을 생성하고
      * When 지하철역 목록을 조회하면
      * Then 2개의 지하철역을 응답 받는다
      */
-    // TODO: 지하철역 목록 조회 인수 테스트 메서드 생성
+    @DisplayName("지하철 목록 조회")
+    @Test
+    void viewStationList(){
+        //given
+        final String STATION_NAME_1 = "강남역";
+        final String STATION_NAME_2 = "숭실대입구역";
+        지하철역_생성(STATION_NAME_1);
+        지하철역_생성(STATION_NAME_2);
+
+        //when
+        ExtractableResponse<Response> getResponse = 지하철역_목록_조회();
+
+        //then
+        List<String> stationsNames = getResponse.jsonPath().getList(KEY_NAME, String.class);
+        assertThat(getResponse.statusCode()).isEqualTo(HttpStatus.OK.value());
+        지하철역_이름_검증(STATION_NAME_1, stationsNames);
+        지하철역_이름_검증(STATION_NAME_2, stationsNames);
+    }
 
     /**
      * Given 지하철역을 생성하고
      * When 그 지하철역을 삭제하면
      * Then 그 지하철역 목록 조회 시 생성한 역을 찾을 수 없다
      */
-    // TODO: 지하철역 제거 인수 테스트 메서드 생성
+    @DisplayName("지하철역 제거")
+    @Test
+    void removeStation(){
+        //given
+        final String STATION = "강남역";
+        ExtractableResponse<Response> createResponse = 지하철역_생성(STATION);
+        long id = createResponse.jsonPath().getLong(KEY_ID);
```

Comment:
```suggestion
        long 강남역 = 지하철역_생성(강남역);
```

생성 된 역의 아이디로 반환하면 테스트가 깔끔해지겠죠?